README.md

```diff
@@ -1 +1,17 @@
-# codacy-test
\ No newline at end of file
+# codacy-test
+
+### Among Codacy’s features
+- Identify new Static Analysis issues
+* Commit and Pull Request Analysis with GitHub, BitBucket/Stash, GitLab (and also direct git repositories)
```

Comment:
Codacy has a fix for the issue: [Lists should be surrounded by blank lines](https://app.codacy.com/gh/franciscodua/codacy-test/pullRequest?prid=7573067)
```suggestion

- Auto-comments on Commits and Pull Requests
```